app/components/dashboard/DcChart.vue

```
@@ -163,7 +163,7 @@
                                                                           v-model="pnl.detail.is_show">詳細&nbsp;</label>
             <label for="ch_pnl_ana" v-show="gg.dt===DT_COVID && pnl.ana.is_chk_show">
                 <input id="ch_pnl_ana" type="checkbox" v-model="pnl.ana.is_show">
-                <i class="fa fa-eye"></i>分析
+                <i class="fa fa-eye"></i>比較分析
```

ESLint

[error] Expected indentation of 14 spaces but found 16 spaces.

(vue/html-indent)